tensorflow_io/core/kernels/avro/parse_avro_kernels.cc

```diff
@@ -204,8 +207,13 @@ Status ParseAvro(const AvroParserConfig& config,
         minibatch_bytes = 0;
       }
     }
-    // 'special logic'
-    const size_t min_minibatches = std::min<size_t>(8, serialized.size());
+    if (avro_num_minibatches_) {
```

Comment:
if avro_num_minibatches_ is not set, what's the default value? can you comment it in the code?

Reply:
Updated